djangocms_versioning/models.py

```diff
@@ -23,49 +20,6 @@ class Version(models.Model):
     state = FSMField(
         default=constants.DRAFT, choices=constants.VERSION_STATES, protected=True)
 
-    def _copy_function_factory(self, field):
-        """
-        Factory for a dynamically defined function that considers the relationship type
-        (1-2-M or M-2-M), when copying related objects between model instances.
-        :param field: Related object.
-        :return: Specific kind of object copy function.
-        """
-        def inner(new):
-            related = getattr(self, field.name)
-            initial = {
-                f.name: getattr(related, f.name)
-                for f in related._meta.fields if not f.auto_created
-            }
-            new_related = related._meta.model(**initial)
-            new_related.save()
-            return new_related
-
-        def inner_m2m(new):
-            related = getattr(self, field.name)
-            related_objects = related.all()
-            return related_objects
-
-        inner_copy = inner_m2m if field.many_to_many else inner
-        return inner_copy
-
-    def _get_relation_fields(self):
-        """Returns a list of relation fields to copy over.
-        If copy_field_order is present, sorts the outcome
-        based on the list of field names
-        """
-        relation_fields = [
-            f for f in self._meta.get_fields() if
-            f.is_relation and
-            f.name not in self.COPIED_FIELDS and
-            not f.auto_created
-        ]
-        if getattr(self, 'copy_field_order', None):
-            relation_fields = sorted(
-                relation_fields,
-                key=lambda f: self.copy_field_order.index(f.name),
-            )
-        return relation_fields
-
     def copy(self):
         """Creates new Version object, with metadata copied over
```

Comment:
Remember to update docstring after you're done with the changes.

Reply:
ok, in the next two PRs ;) And remember to remind me ;)